StellarApp/AppDelegate.swift

```diff
@@ -8,11 +8,12 @@
 
 import UIKit
 
+let appDelegate = UIApplication.shared.delegate as! AppDelegate
 @UIApplicationMain
 class AppDelegate: UIResponder, UIApplicationDelegate {
     
     var window: UIWindow?
-    
+    var deviceOrientation = UIInterfaceOrientationMask.landscape
     
```

Comment:
do we need to set up for the landscape? why im asking is because usually iPhone is automatically turn the phone side